src/crsf/crsf.cpp

```diff
@@ -4,6 +4,7 @@
 #include "protocol.h"
 #include <chrono>
 #include <thread>
+#include <algorithm>
```

Comment:
Are these changes adding headers + removing `constexpr` necessary for compilation on your machine? If they are then they should be put into a separate commit. Something like `treewide: Fix compilation on <compiler>`.

Reply:
Yes, it's just for me. I didn't plan to include them as part of the PR.